independent-projects/arc/runtime/src/main/java/io/quarkus/arc/InjectableBean.java

```diff
@@ -90,7 +90,7 @@ default String getName() {
         return Collections.emptySet();
     }
 
-    @Override
+    // Deprecated method which can be safely removed once we use CDI 4.0+
```

Comment:
Should we deprecate those methods as well?

Reply:
In what way? Adding `@Deprecated` or logging something?
Annotation won't help much as people are coding against CDI API and that one doesn't (for instance) deprecate `BM#fireEvent()` with annotation; it merely mentions that in javadoc :-(

Reply:
You're right that it would not help users... but it would align with our deprecation strategy, plus get warnings in IDE and from the compiler. Anyway, +0 from me.

Reply:
I'd add `@Deprecated` anyway, since we often point people towards the `Arc` entrypoint, from which you usually get to the ArC-specific types.

Reply:
Fair enough, marked all three methods as `@Deprecated`